src/test/java/com/coravy/hudson/plugins/github/GithubLinkAnnotatorTest.java

```diff
@@ -25,13 +44,44 @@ public final void testAnnotateStringMarkupText() {
                         + "issues/9876/find'>close #9876</a> link");
     }
 
+    @Test
+    public final void testChangeSetAnnotateStringMarkupText() {
```

Comment:
Thanks for the clarity.  As time allows, I'll rework this pull request using Hamcrest assertions and will resubmit.  I'll close this pull request and replace it with a new pull request when the Hamcrest based assertions are ready.